api/v2/migrations/dropTables.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+import { Client } from 'pg';
+import { dropTables } from './dbQuery';
+
+const config = {
+  user: 'postgres',
+  database: 'teamworkdb',
+  password: 'postgres',
+  port: 5432,
+  max: 10,
+  idleTimeoutMillis: 30000,
+};
```

Comment:
This config object data shouldn't be accessible to the public move it to the dotenv file.